### DISCUSSION TITLE
How to write a GeoParquet file?

### DISCUSSION THREAD

**Discussion by nshiab** [duckdb/duckdb-spatial#360] [Q&A] [answered]
Hi!

You have an example of how to write data to a GeoJSON file. Would it be possible to add an example of how to write data to a GeoParquet file?

Thank you for your awesome work. 🙏

**Comment by cholmes** [comment, marked as answer]
As of DuckDB 1.1 (though use 1.1.1 as it got a bug fix that may help) if you have the spatial extension on (`install spatial; load spatial`) and your data is geometries then you just need to use the normal Parquet output:

```
COPY <my_table> TO '<my_output>.parquet' WITH (FORMAT PARQUET);
```

You can set the compression easily, zstd works great for spatial data (brotli gets even more compressed, but the compression in duckdb seems to take a good bit longer):
```
COPY <my_table> TO '<my_output>.parquet' WITH (FORMAT PARQUET, COMPRESSION 'zstd');
```

**Reply by nshiab** [reply]
Thank you @cholmes !
